include/deal.II/base/function_parser.h

```diff
@@ -47,6 +47,20 @@ template <typename> class Vector;
  * used in the step-33 and step-36 tutorial programs (the latter being much
  * simpler to understand).
  *
+ * In addition to the built-in functions of muparser, namely
+ * @code
+ * sin, cos, tan, asin, acos, atan, sinh, cosh, tanh, asinh, acosh, atanh,
+ * atan2, log2, log10, log, ln, exp, sqrt, sign, rint, abs, min, max, sum, avg
+ * @endcode
+ * this class also supports:
+ * - <tt>if(condition, then-value, else-value)</tt>
+ * - <tt>|</tt> and <tt>&</tt> (logic or and and)
```

Comment:
These are logic operators implemented with `||` and `&&` (by first converting their arguments to ints).